collections/_article/rendering-priorities-july-2023.md

```diff
@@ -35,6 +35,8 @@ This is, in part, a carry-over from the last post. At first our idea was to add
 
 Bastiaan worked super hard on this and had both systems nearly ready for 4.1, but unfortunately didn't complete the work in time to be included with 4.1's release as more testing is needed. We expect this work will be merged soon and will be available for testing and feedback prior to 4.2's release. We ended up needing to introduce a few more settings than expected, so we really need community feedback to make sure the system is intuitive and helpful.
 
+_edit: In testing we have identified performance issues that stop us from merging this work. Accordingly, this will not make it into 4.2. The work is ongoing and hopefully we will have something we are happy with ready for a later release._
```

Comment:
```suggestion
_September 2023 edit: In testing we have identified performance issues that stop us from merging this work. Accordingly, this will not make it into 4.2. The work is ongoing and hopefully we will have something we are happy with ready for a later release._
```
So that if someone goes back to look at this post and wondered why the feature didn't make it, they have a bit of an idea of the timeline.

Reply:
Good idea